BFF/v3/docs/themes/hugo-theme-learn/layouts/partials/logo.html

```diff
@@ -0,0 +1,3 @@
+<a id="logo" href="{{ .Site.BaseURL }}">
+  <img src="/bff/v2/images/logo.png" />
```

Comment:
This logo should point to v3 so it works locally

Reply:
fixed